content/applications/websites/livechat.rst

```diff
@@ -242,51 +271,71 @@ If a users :guilabel:`Online Chat Name` is not set, the name displayed will defa
       :align: center
       :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
 
+Set online chat languages
+-------------------------
+
+If an operator speaks multiple languages, they can add this information to their user profile. This
+information can then be used to determine what conversations the operator is to be assigned. A
+visitor's language is determine via website settings.
+
+To add one or more :guilabel:`Online Chat Language`, click on the user avatar in the top-right
```

Comment:
again,  since this field isn't technically being "seen" at this point - I think the use of ** should replace the GUI label here.